### PR TITLE
es_es update

### PR DESCRIPTION
**Pull request by ghost**
Includes spelling corrections.

### CODE REVIEW

**Comment by BluSunrize** [comment]
How does this integrate with #4838?

@docanuto and @FrannDzs you might want to coordinate here, since it's two spanish translations.

**Comment by ghost** [comment]
> How does this integrate with #4838?
> 
> @docanuto and @FrannDzs you might want to coordinate here, since it's two spanish translations.

From what I have seen the other translation is misspelled and wrong capitalized. Is unchecked.

Also, I think that Frann is argentinian. So the translation probably is more accurate es_ar.

**Comment by FrannDzs** [comment]
Hi guys @docanuto @BluSunrize  I am the one who did the last translation of this mod including the manual from scratch. There were some things misspelled by me, but if I check your translation there are misspelled things too. For example, Wolf Pack is not translated literally and the industrial harvester is not a wheelbarrow. Anyway, I like your last translation, we could review it and take it as a base now. You improved some contexts and the truth is that they are very good.

I'm Argentinian, but I'm not looking to Argentinize, I want a more neutral Spanish, es_es is used as neutral Spanish in Spanish-speaking countries, most of the mods don't have Spanish translation in Argentina or Uruguay, for example, so we should make it as neutral as possible, that's why I use words from Spain and others from America.

How do we do to agree? :)

File: src/main/resources/assets/immersiveengineering/lang/es_es.json [hunk 1]
```diff
@@ -1,1207 +1,1207 @@
...
+  "block.minecraft.banner.immersiveengineering_hammer.silver": "Martillo gris claro",
+  "block.minecraft.banner.immersiveengineering_hammer.gray": "Martillo gris",
+  "block.minecraft.banner.immersiveengineering_hammer.pink": "Martillo rosa",
+  "block.minecraft.banner.immersiveengineering_hammer.lime": "Martillo verde lima",
+  "block.minecraft.banner.immersiveengineering_hammer.yellow": "Martillo amarillo",
+  "block.minecraft.banner.immersiveengineering_hammer.lightBlue": "Martillo azul claro",
+  "block.minecraft.banner.immersiveengineering_hammer.magenta": "Martillo magenta",
+  "block.minecraft.banner.immersiveengineering_hammer.orange": "Martillo naranja",
+  "block.minecraft.banner.immersiveengineering_hammer.white": "Martillo blanco",
+  "block.minecraft.banner.immersiveengineering_bevels.black": "Martillo negro",
```
Comment (FrannDzs) [inline src/main/resources/assets/immersiveengineering/lang/es_es.json]: Here is an error

**Comment by FrannDzs** [comment]
> > How does this integrate with #4838?
> > @docanuto and @FrannDzs you might want to coordinate here, since it's two spanish translations.
> 
> From what I have seen the other translation is misspelled and wrong capitalized. Is unchecked.
> 
> Also, I think that Frann is argentinian. So the translation probably is more accurate es_ar.



> > How does this integrate with #4838?
> > @docanuto and @FrannDzs you might want to coordinate here, since it's two spanish translations.
> 
> From what I have seen the other translation is misspelled and wrong capitalized. Is unchecked.
> 
> Also, I think that Frann is argentinian. So the translation probably is more accurate es_ar.

Hey friend, I understand your effort, but notice that your translation is outdated, it is missing "strings", again, I love your translation, just that you should make a fork from me, so you can use my template that I have already added the lines, or as you like!

**Comment by FrannDzs** [comment]
@docanuto do you have whatsapp or e-mail so we can communicate better?

**Comment by ghost** [comment]
> @docanuto do you have whatsapp or e-mail so we can communicate better?

Im actually in the official Discord. :)

**Comment by FrannDzs** [comment]
@docanuto we should make a mix of the two translations, you mark me errors and suggestions and I mark you the same, and from there we get an improved one, what do you think? start by making the fork of Immersive Engineering from my account, which is already updated, check my changes please, greetings.

> > @docanuto do you have whatsapp or e-mail so we can communicate better?
> 
> Im actually in the official Discord. :)

I'm nekrodamus on discord now

**Comment by ghost** [comment]
Done. :)

**Comment by FrannDzs** [comment]
An urgent upgrade is needed before this is merged.

**Pull request closed without merging**